base/shared/src/main/scala/scalaz/data/Fix.scala

```diff
@@ -0,0 +1,27 @@
+package scalaz
+package data
+
+import Prelude._
+import typeclass.Liskov
+import typeclass.Liskov.<~<
+
+trait FixModule {
+  type Fix[F[_]]
+
+  /* functions */
+
+  def   fix[F[_]](f: F[Fix[F]]):   Fix[F]
+  def unfix[F[_]](f:   Fix[F] ): F[Fix[F]]
+
+  def liftLiskov[F[_], G[_]](ev: ∀[λ[α => F[α] <~< G[α]]])(implicit F: IsCovariant[F]): Fix[F] <~< Fix[G]
+}
+
+private[data] trait FixImpl extends FixModule {
+  type Fix[F[_]] = F[Any]
```

Comment:
[This can be safer.](https://github.com/TomasMikula/scalaz/compare/IList...S11001001:IList+Fix)

Reply:
Nice! But I wanted to make `data.Fix` an `object` instead of a `val`, due to performance concerns with `invokevirtual` raised in #1450, where I used a `val`.

Reply:
The problem with that approach is, as of e5566352

```scala
scala> List(1, 2, 3): scalaz.data.Fix[List]
res0: scalaz.data.Fix[List] = List(1, 2, 3)

scala> def conforms[A, B >: A] = 'yep
conforms: [A, B >: A]=> Symbol

scala> conforms[scalaz.data.IList[String], scalaz.data.IList[Any]]
res2: Symbol = 'yep

scala> conforms[Option[(Int, Any)], scalaz.data.IList[Int]]
res6: Symbol = 'yep
```

Reply:
Ah, thanks for checking this! I had a suspicion we can't trust Scala with keeping `private` things private. I will incorporate your suggestion then.